### ISSUE TITLE
Parentheses out of layer names

### ISSUE THREAD

**Issue by nick4rivers** [Riverscapes/QRiS#326]
Please get rid of the () in the Active Channels and Centerlines layer names. It's inconsistent with all the rest of the layer names and is asking for trouble when writing calculations. Come on people....is this amateur hour.

![Screen Shot 2024-01-24 at 3 29 49 PM](https://github.com/Riverscapes/QRiS/assets/34071692/fdcb12c4-d50a-4d0f-831b-5df47d661c0d)

**Comment by joewheaton** [comment]
You are right on inconsistency. However, there is something on plural vs. singular here. We don't need the `(bankfull)`.  And everywhere else (e.g. `channel junctions`) even though there could be one. This is not for @KellyMWhitehead (I don't think). I believe @KellyMWhitehead is getting this from the QML so this is a curation issue in our court and @jdgilby should be able to fix. I am assigning to him and moving to other repo. @KellyMWhitehead please correct me if I am wrong. 

- [ ] Change to `Active Channels`
- [ ] Change to `Centerlines of Active Channels`
